### ISSUE TITLE
Modelul este re-inițializat în fiecare epocă

### ISSUE THREAD

**Issue by danoneata** [Kronus97/normalizing-flows-digits#1]
Hello! O potențială problemă pe care o văd este că modelul și optimizatorul sunt re-inițializate la fiecare iterație prin epocii (deci antrenarea o ia de la capăt la fiecare epocă):

```python
def train(x, num_iter, train_loader, weight_decay=None):
    num_dim = x.shape[1]
    flow = build_flow(num_dim)
    ...

for epoch in range(1, epochs + 1):
    model = train(train_x, 5000, trainloader)
```

**Comment by Kronus97** [comment]
Ok. Mulțumesc de observație. Am să modific.
